scripts/_common.sh

```diff
@@ -5,10 +5,8 @@
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies="mongodb mongodb-server mongo-tools"
-pkg_dependencies_buster="mongodb-org mongodb-org-server mongodb-org-tools"
-mongodb_stretch="mongodb"
-mongodb_buster="mongod"
+pkg_dependencies=""
+
 nodejsversion=12.16.3
```

Comment:
Don't we have to use only major version ? 

```suggestion
nodejsversion=12
```

Reply:
no we don't ^^:

https://github.com/YunoHost/yunohost/blob/d55fec4044cc8f2ee388aea4e17c28568e589cab/data/helpers.d/nodejs#L98

We should prefer to use major version number, but it's not mandatory